.github/workflows/ci.yml

```diff
@@ -64,4 +64,4 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Unit Race Tests
-        run: go test -timeout 120s -race -gcflags=all=-d=checkptr=0 ./...
+        run: go test -timeout 240s -race -gcflags=all=-d=checkptr=0 ./...
```

Comment:
Not nice :( Does that mean our CI runs twice as long now?

Update:
I compared the running time of a CI run before and after the update. They are roughly equal. I don't think we need to bump the timeout, do we?

Reply:
The i dont know why it time-outed once… I will try it without again.